core/Url.php

```diff
@@ -916,4 +916,18 @@ public static function addCampaignParametersToMatomoLink(
         $pathAndQueryString = UrlHelper::getPathAndQueryFromUrl($url, $newParams, true);
         return 'https://' . $domain . '/' . $pathAndQueryString;
     }
+
+    /**
+     * Create an external link tag with optional campaign params if link goes to matomo.org
+     */
```

Comment:
```suggestion
     * @since 5.6.0
     */
```

If this method is meant to be used by plugins, then we should add a hint since which version the method is available. This makes it easier for plugin developer to know which version their plugin would be compatible with.

This also means we should avoid using this method in plugins. that are not shipped with core, before Matomo 6, as we otherwise would have to increase the minimum required Matomo version.

Reply:
So keep it in TagManager and reset Provider and CustomVariables? Can do that.

Reply:
Yes. I don't think it makes sense to open PRs for the other plugins yet. We can consider doing that with Matomo 6.